packages/headless/src/controllers/standalone-search-box/headless-standalone-search-box.ts

```diff
@@ -51,6 +52,11 @@ export interface StandaloneSearchBox extends SearchBox {
    */
   submit(): void;
 
+  /**
+   * Reset a standalone search box's state. To be dispatched on SPAs after the redirection has been triggered.
```

Comment:
```suggestion
   * Resets the standalone search box state. To be dispatched on SPAs after the redirection has been triggered.
```
Curious: what's `SPA` here?

Reply:
single page application, i'll type it out in the docs too

Reply:
Thanks, it'll help some users I think 🙂